### ISSUE TITLE
Unquoted service path

### ISSUE THREAD

**Issue by Mohad0** [freelan-developers/freelan#234]
Hi FreeLan team,

After installing FreeLan 2.2.0 from:
- https://github.com/freelan-developers/freelan/releases/download/2.2/freelan-2.2.0-x86-install.exe
- https://github.com/freelan-developers/freelan/releases/download/2.2/freelan-2.2.0-amd64-install.exe

I noticed that its service is hijackable due to the unquoted service path. Using this vulnerability, attackers can execute different files as `FreeLAN Service`. It allows local users to replace the service with arbitrary code to escalate their privileges. I hope you check this link for more details: https://cwe.mitre.org/data/definitions/428.html

**Comment by M507** [comment]
```
C:\Users\Administrator>sc qc "FreeLAN Service"
[SC] QueryServiceConfig SUCCESS

SERVICE_NAME: FreeLAN Service
        TYPE               : 10  WIN32_OWN_PROCESS
        START_TYPE         : 2   AUTO_START
        ERROR_CONTROL      : 1   NORMAL
        BINARY_PATH_NAME   : C:\Program Files\FreeLAN\bin\freelan.exe
        LOAD_ORDER_GROUP   :
        TAG                : 0
        DISPLAY_NAME       : FreeLAN Service
        DEPENDENCIES       : tap0901
                           : Dhcp
        SERVICE_START_NAME : LocalSystem

C:\Users\Administrator>
```

**Comment by TJ-59** [comment]
Yup, noticed too, I submitted a Pull Request for this purpose, now we just need the team to have a look at it whenever they have time.
Meanwhile, interested people can have a look at #245 or my github for a solution.
